.travis.yml

```diff
@@ -5,13 +5,8 @@ os:
   - osx
   - windows
 julia:
-  - 1.0
-  - 1.2
+  - 1.3
```

Comment:
Why test on 1.3? I'll suggest using `- 1` here such that it'll pick up the latest release.